src/lib/driver/command.ts

```diff
@@ -0,0 +1,6 @@
+export enum DriverCommand {
+  updateLogConfig = "driver.update_log_config",
+  getLogConfig = "driver.get_log_config",
```

Comment:
Would be nice to create some generic "getConfig" command to get all driver info at once.

Reply:
What other information do you want to get about the driver? Right now we don't model the state of the driver at all in the library

Reply:
We can start with what we have:

```ts
interface Driver {
  logConfig: LogConfig;
  statisticsEnabled: boolean;
}
```

Reply:
(ofc we should mimic the names of the properties on the `driver` object)

Reply:
When we have only one client, this can be done easily because we can update the driver state when we enabled/disable statistics or update the log config after getting the initial state. When we have multiple clients, maintaining this state down to the lib would require new events to be passed to all clients whenever those commands are called.

Do we want to maintain this state down to the library, or is the goal here just to maintain the state of the driver in the server for now?

Reply:
Let's not add things to the lib unless we are going to use it.